extensions/amp-auto-ads/0.1/ad-network-config.js

```diff
@@ -192,10 +192,15 @@ class AdSenseNetworkConfig {
 
   /** @override */
   getAttributes() {
-    return dict({
+    const attributesObj = dict({
```

Comment:
Does it matter if an attribute value is null or undefined?  If not, this could be simplified to:

return dict({'type': 'adsense', 'data-ad-client': this.autoAmpAdsElement_.getAttribute('data-ad-client'), 'data-ad-host': this.autoAmpAdsElement_.getAttribute('data-ad-host')});

Reply:
it cannot be null/undefined. this was actually my initial change. tagged you in the thread that explains why

Reply:
it would be nice if we could pass inside dict() function call Object.assign, since it skips over null params. But the dictionary function call only accepts object literals.